GetIntoTeachingApi/Models/BaseModel.cs

```diff
@@ -188,7 +188,8 @@ private void NullifyInvalidFieldAttributes(IValidatorFactory validatorFactory)
                 return;
             }
 
-            var result = validator.Validate(this);
+            var context = new ValidationContext<object>(this);
```

Comment:
Could this `object` be a `BaseModel`?

Reply:
Yep good catch I'll update that and squash it into the relevant commit